src/website/views/templates/fieldset.pug

```diff
@@ -0,0 +1,218 @@
+---
+title: Fieldset
+---
+
+p.-text Basic example of several types of form elements in one fieldset.
+
+.example.-mb--3
+  .chi-app.-py--3.-bg--grey-20(style="border-radius:.15em .15em 0 0;")
+    .a-grid__container
+      .a-grid.-noGutter.-b--1.-p--3.-bg--white
```

Comment:
Can you remove `-b--1` and `-bg--white` and replace with `a-card`?

Reply:
can you add -flex--row class too?

Reply:
done